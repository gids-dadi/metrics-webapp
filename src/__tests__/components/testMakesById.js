import renderer from 'react-test-renderer';
import GetMakesByName from '../../__Mocks__/getMakesByName';

it('component render correctely', () => {
  const tree = renderer
    .create(<GetMakesByName />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
