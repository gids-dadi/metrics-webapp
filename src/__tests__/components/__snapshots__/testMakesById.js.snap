// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component render correctely 1`] = `
<div
  className="child-page"
>
  <img
    alt=""
    className="big-car-img"
    src="black-car8.png"
  />
  <div
    className="line"
  />
  <h2>
    Vehicles Makes by Name
  </h2>
  <p>
    Enter the name of vehicle
  </p>
  <input
    onChange={[Function]}
    onKeyDown={[Function]}
    placeholder="...search"
    type="search"
  />
  <section
    className="container-cards"
  />
  <div
    className="buttons"
  >
    <h4>
      1
       
      of
       
      0
    </h4>
    <button
      className="previous"
      onClick={[Function]}
      type="button"
    >
      previous
    </button>
    <button
      className="next"
      onClick={[Function]}
      type="button"
    >
      next page
    </button>
  </div>
</div>
`;
